### PR TITLE
Allow passing parameters to forgit::reset::head

### PR DESCRIPTION
**Pull request by sandr01d**
## Check list

- [X] I have performed a self-review of my code
- [ ] I have commented my code in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Description

Allow passing parameters to forgit::reset::head in the same way we already do for most other functions.

## Type of change

- [ ] Bug fix
- [X] New feature
- [ ] Refactor
- [ ] Breaking change
- [ ] Documentation change

## Test environment

- Shell
    - [X] bash
    - [X] zsh
    - [X] fish
- OS
    - [X] Linux
    - [ ] Mac OS X
    - [ ] Windows
    - [ ] Others:

### CODE REVIEW

**Review by carlfriedrich — CHANGES_REQUESTED** [review]

**Comment by sandr01d** [comment]
Must have overlooked that, good catch! I've made changes accordingly.

File: bin/git-forgit
Comment (carlfriedrich) [inline bin/git-forgit:1]: Great, thanks for the update, but now you missed the argument. :-)

**Review by carlfriedrich — CHANGES_REQUESTED** [review]

**Comment by sandr01d** [comment]
Guess I should code when tired. Third time's the charm :)

**Comment by carlfriedrich** [comment]
> Guess I shouldn't code when tired. Third time's the charm :)

That's what reviews are for. :-)

Great addition, thanks a lot for your contribution!

**Pull request merged**